docs/reference/reference/gel_toml.rst

```diff
@@ -56,9 +141,22 @@ Example
     [project]
     schema-dir = "db/schema"
 
+    [hooks]
+    project.init.after="setup_dsn.sh"
+    branch.wipe.after=""
+    branch.switch.after="setup_dsn.sh"
+    schema.update.after="gel-orm sqlalchemy --mod compat --out compat"
```

Comment:
tiny nit: can we format these lines the same as the other lines:

```suggestion
    project.init.after = "setup_dsn.sh"
    branch.switch.after = "setup_dsn.sh"
    schema.update.after = "gel-orm sqlalchemy --mod compat --out compat"
```
Also, was there a reason we showed an empty hook?